### DISCUSSION TITLE
Spinners are not affected by AR the same way other hit objects are.

### DISCUSSION THREAD

**Discussion by Toasterly** [ppy/osu#25641] [Ideas]
In standard spinners appear at the same time no matter which AR a map is played at, this can cause a hit object meant to be hit after a spinner to show up before the spinner is visible on your screen at lower ARs.

It feels unintuitive and weird to have objects appearing out of order when playing standard, having the spinner show earlier or later depending on AR like other hit objects would make readability around spinners much more logical for lower/higher AR players.

**Comment by peppy** [comment]
is this different from stable?

**Reply by Toasterly** [reply]
Both Lazer & Stable do not account for AR when showing spinners to my knowledge, apologies for not being more clear in my wording. This is a suggestion for a change to Lazer to make all objects appear in the correct order even in tight edge cases. Perhaps you could put it behind a toggle in settings for those who want the new implementation? I was hoping this would come up in some community meeting, discussion, or PR somewhere but I'm seemingly one of the few people in a very niche gameplay category who even notices it.

For some context I am an EZ mod player and will often take AR below 0 using the difficulty adjust mod. I primarily use Lazer's release now for playing rather than Stable because it is a better experience in every way.
